contracts/pcv/maker/MakerRouter.sol

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {IMakerRouter} from "./IMakerRouter.sol";
+import {IDSSPSM} from "./IDSSPSM.sol";
+import {IPegStabilityModule} from "../../peg/IPegStabilityModule.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {CoreRef} from "../../refs/CoreRef.sol";
+import {TribeRoles} from "../../core/TribeRoles.sol";
+import {Constants} from "../../Constants.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+
+/// @notice This contracts allows for swaps between FEI-DAI and FEI-USDC
+/// by using the FEI FEI-DAI PSM and the Maker DAI-USDC PSM
+/// @author k-xo
+contract MakerRouter is IMakerRouter, CoreRef {
+    using SafeERC20 for IERC20;
+
+    /// @notice reference to the Maker DAI-USDC PSM that this router interacts with
+    /// @dev points to Makers DssPsm contract
+    IDSSPSM public immutable daiPSM;
```

Comment:
Would maybe make it clear in the variable name that is is the `makerDaiPSM` and avoid any confusion in it possibly being a Volt created PSM

Reply:
The type and comment makes it very clear